tests/core/interceptor.c

```diff
@@ -56,7 +56,9 @@ TESTLIST_BEGIN (interceptor)
   TESTENTRY (already_replaced)
 # ifndef HAVE_ASAN
   TESTENTRY (replace_one)
+# if defined (HAVE_FRIDA_GLIB)
   TESTENTRY (replace_two)
+# endif
 # endif
```

Comment:
```suggestion
#endif
```

(Just spotted this existing style bug.)